frontend/public/learn/Interactive-UI-Elements.elm

```diff
@@ -132,36 +141,26 @@ pattern used by interactive UI elements in Elm:
   [box]: http://library.elm-lang.org/catalog/elm-lang-Elm/latest/Graphics-Input#checkbox
 
 ```haskell
-checkbox : Handle a -> (Bool -> a) -> Bool -> Element
-checkbox handle processingFunction checked = ...
+checkbox : (Bool -> Signal.Message) -> Bool -> Element
+checkbox handler checked = ...
 ```
 
-The three arguments work like this:
+The two arguments work like this:
 
-  1. Clicking on a checkbox generates an event, and the handle specifies which
-     `Input` these events should be sent to. The generated event is a boolean
+  1. Clicking on a checkbox generates an event, and the handler specifies which
+     `Channel` these events should be sent to. The generated event is a boolean
      value that represents what the user *wants* the checkbox to be. Clicking an
      unchecked box generates a `True` event, and clicking a checked box generates
      a `False` event.
```

Comment:
I think this needs to be reworked a little more. It's not really a handler anymore, it's a function to create a message given the (new) state of the UI element. A message encapsulates a channel, and a message derived from the new state. Also, line 145 is weird: it's a library function and we shouldn't really be referring to its parameters. Perhaps name them on like 148?